### PR TITLE
Hw05 branch

### PR DESCRIPTION
**Pull request by samanthashepard**

### CODE REVIEW

**Comment by samanthashepard** [comment]
@eburkey @tylerallan

File: Homeworks/Homework 5/hw05.Rmd [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  filter(Date > as.Date("2019-01-01") & Date < "2020-12-31") %>% 
+  ggplot() +
+  aes(x = as.Date(Date), y = noscent, color = Scented_vs_Unscented) +
+  geom_point(size = .9, alpha = .05) +
+  geom_vline(xintercept = as.numeric(as.Date("2020-01-20")), linetype = "dashed") +
+  scale_x_date(name = "Date", date_breaks = "3 months", date_labels = "%m-%Y") +
+  scale_y_continuous(name = "Proportion of 'No Smell' Reviews") +
+  stat_smooth(method = "loess")
+
+
```
Comment (eburkey) [inline Homeworks/Homework 5/hw05.Rmd:223]: Great job! Your code is really easy to read and everything is quite organized. I do think that in some areas you could use a different code such as when you do your group by code, but that could also be a preference thing. 
----------------------------------------------------
Topic                       | Excellent | Satisfactory | Needs Work |
----------------------------|-----|-----|-----|
**Coding style**            |  | :heavy_check_mark: |  |
**Coding strategy**         |  | :heavy_check_mark: |  |
**Presentation: graphs**    | :heavy_check_mark: |  |  |
**Presentation: tables**    | :heavy_check_mark: |  |  |
**Achievement, creativity** | :heavy_check_mark: |  |  |
**Ease of access**          | :heavy_check_mark: |  |  |

File: Homeworks/Homework 5/hw05.Rmd [hunk 2]
```diff
@@ -0,0 +1,229 @@
+---
+title: "Consequences of Covid-19 for Candle Critiques"
+author: "Your Name Here"
```
Comment (tylerallan) [inline Homeworks/Homework 5/hw05.Rmd:3]: Might want to include your name in the YAML code here

File: Homeworks/Homework 5/hw05.Rmd [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+date: "3/22/2021"
+output: html_document
```
Comment (tylerallan) [inline Homeworks/Homework 5/hw05.Rmd:5]: Make sure to fold the code blocks, enable code highlighting, and set a theme here.

File: Homeworks/Homework 5/hw05.Rmd [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+  4. A dashed vertical line at January 01, 2020
+    1. Hint: use `geom_vline()` and set `xintercept = as.numeric(as.Date("2020-01-20"))`
+    
+Be sure to make the plots beautiful.
+
+## Scented candle product ratings: January 2017 - January 2020 
+
+```{r}
+scented %>% 
+  select (CandleID, Date, Rating, Review) %>%
```
Comment (tylerallan) [inline Homeworks/Homework 5/hw05.Rmd:63]: You don't need to worry about selecting out variables here they will be included when you call the dataset in the beginning.

**Comment by tylerallan** [comment]
Peer Review of HW-05 for Samantha Shepard
----------------------------------------------------
Topic                       | Excellent | Satisfactory | Needs Work |
----------------------------|-----|-----|-----|
**Coding style**            | :heavy_check_mark: |  |  |
**Coding strategy**         |  | :heavy_check_mark: |  |
**Presentation: graphs**    |  :heavy_check_mark:  |  |  |
**Presentation: tables**    | :heavy_check_mark:  |   |  |
**Achievement, creativity** | :heavy_check_mark: |  |  |
**Ease of access**          | :heavy_check_mark: |  |  |

Great work Sam! I've added a couple of comments throughout the code of things you may want to look at but otherwise good job!

**Pull request merged**